app/src/userpages/components/StreamPage/Show/KeyView/index.jsx

```diff
@@ -47,9 +49,19 @@ export class KeyView extends Component<Props> {
         }
     }
 
-    getKeys = () => {
-        if (this.props.disabled || this.props.streamId == null) { return }
-        this.props.getKeys(this.props.streamId)
+    getKeys = async () => {
+        const { disabled, streamId, getKeys } = this.props
+
+        if (disabled || streamId == null) { return }
+        try {
+            await getKeys(streamId)
+        } catch (error) {
```

Comment:
Should it show this notification if the component is unmounted?

Reply:
Fixed in f1617c673af45afc5d8ad3854c59f86f7432d7a4.